OKTA-529338 Update Publish an OIN integration with new Support section (#3640)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-app/main/index.md

After you have built a functioning app integration, a few steps are required to submit it to Okta for review and publication in the OIN:

1. Create a [customer-facing configuration guide](#prepare-a-customer-facing-configuration-guide).
    >**Note:** This guide is required for SCIM and OIDC OIN apps. It's optional for SAML integrations, as Okta provides its own documentation for those apps. The guide is supplied with your published app.

> 1. [Submit your integration](#submit-an-integration) to Okta through the OIN Manager tool. Your submission must provide Okta with the general and protocol-specific metadata that is required to create a customized integration for publication in the OIN.

Reviewer: ```suggestion
1. [Submit your integration](#submit-an-integration) to Okta through the OIN Manager tool. Your submission must provide Okta with the general and protocol-specific metadata that's required to create a customized integration for publication in the OIN.
```

  >**Note:** SWA app integrations are no longer accepted for publication in the OIN catalog. However, existing SWA apps are still maintained by the OIN team.

### Understand the submission review process

> The submission review process begins after you click **Submit for Review** in the [OIN Manager](https://oinmanager.okta.com). Okta sends you an email notification that your integration is now queued for review by the Okta OIN team, along with a date by which you should complete the initial review.

Reviewer 2: maybe: "...begins after you click..." > "...begins when you click..." ?

Would it be inferred that the reader will be the recipient of the notification? In that case we could simplify to:
"Okta sends an email notification that your integration..."

Also, I don't think that the reader needs to complete the initial review? Isn't the OIN team doing the review? If so, maybe something like:
"...the Okta OIN team, which includes the date that the initial review of the integration is expected to be complete."

Author: Thanks Tim. I'll implement all of your suggestions, except the "Okta sends you an email notification" (I wanted to be clear that the recipient of the notification is the person who submitted the integration).

> The submission review process begins when you click **Submit for Review** in the [OIN Manager](https://oinmanager.okta.com). Okta sends you an email notification that your integration is now queued for review by the Okta OIN team, which includes the date that the initial review of the integration is expected to be completed.

Reviewer: ```suggestion
The submission review process begins when you click **Submit for Review** in the [OIN Manager](https://oinmanager.okta.com). Okta sends you an email notification that your integration is now queued for review by the Okta OIN team, which includes the date that the initial review of the integration is expected to finish.
```

> The OIN Manager always shows the current status of your integration.

Reviewer 2: Is 'always' necessary? 
Your watch shows the current time.
Your watch always shows the current time.

#### Step 1: Initial Review

* **Pending review by Okta**: The Okta OIN team is notified of your submission. Okta reviews the submission and notifies you by email when the submission review is complete.

> * **Action required**: Okta has reviewed your submission and found issues that require your attention. Check your email for results from the Okta initial review. Sign in to OIN Manager, update the requested details, and click **Submit for Review**. After the submission review is complete, the Okta OIN team moves it to Step 2 for QA testing.

Reviewer 2: Maybe:
"After the submission review is complete..."
>
"After submitting your changes, the Okta OIN team moves your submission to Step 2 for QA testing."
or maybe
"After submitting your changes, the Okta OIN team moves your submission to step 2."

Author: I'm updating it to:

> After the OIN team reviews your updated submission and verifies that the issues have been resolved, the submitted integration moves to step 2 for QA testing.

let me know what you think?

> * **Action required**: Okta has reviewed your submission and found issues that require your attention. Check your email for results from the Okta initial review. Sign in to OIN Manager, update the requested details, and click **Submit for Review**. After the OIN team reviews your updated submission and verifies that the issues have been resolved, your submission moves to step 2 for QA testing.

Reviewer: ```suggestion
* **Action required**: Okta has reviewed your submission and found issues that require your attention. Check your email for results from the Okta initial review. Sign in to OIN Manager, update the requested details, and click **Submit for Review**. After the OIN team reviews your updated submission and verifies that the issues are resolved, your submission moves to step two for QA testing.
```

> #### Step 2: Code Review

Reviewer: ```suggestion
#### Step 2: Code review
```

> * **Pending review by Okta**: The Okta OIN team is now conducting internal QA tests and notifies you by email when the QA review is complete. If the QA test is successful, your submission is published automatically in the OIN.

Reviewer 2: Normally active voice, but here maybe:
"...and notifies you by email when..." would be better as "...and will notify you by email when..."?

I might swap "published automatically" to "automatically published"

Author: Implemented you suggestion with slight changes:
The Okta OIN team conducts internal QA tests and will notify you by email when the QA review is complete. If the QA test is successful, your submission is automatically published in the OIN.

Author: I'm keeping the active voice to correspond with step 1:
The Okta OIN team conducts internal QA tests and notifies you by email when the QA review is complete. If the QA test is successful, your submission is automatically published in the OIN.

* **Action required**: Okta has found QA issues that require your correction. Check your email for results from the Okta QA review. Make the requested changes as an update to your existing submission.

> * **Final review by Okta**: The Okta OIN team is conducting a final internal QA test based on previously requested changes and notifies you by email when the final QA review is complete. If the review is successful, your submission is published automatically in the OIN.

Reviewer 2: if we do the change from "notifies you" to "will notify you" in the previous section, then make the change here as well. Otherwise, no change.

Similarly for published automatically/automatically published.

#### Step 3: Published

* Congratulations, your integration is published in the OIN!

The following flowchart outlines the entire process:

<div class="three-quarter">

![ISV Submission process flow](/img/oin/isv-portal_submission_flow.png)

</div>

### Submission support

Getting your app integration in the OIN catalog involves two phases: creating a functional integration and submitting it through the OIN publication process. For each phase in the process, Okta has an associated support stream to assist you.

When you are constructing your Okta integration, you can post a question on the [Okta Developer Forum](https://devforum.okta.com/).

> When you need help during the submission process, use the Support section in the App Integrations page after you sign in to the [OIN Manager](https://oinmanager.okta.com). This section provides you with the following resources from the [Okta developer portal](https://developer.okta.com/):

Reviewer 2: "in the App Integrations page" > "of the App Integrations page"

"provides you with the following" > "provides the following"

> When you need help during the submission process, use the Get Support section of the My App Integrations page after you sign in to the [OIN Manager](https://oinmanager.okta.com). This section provides the following resources from the [Okta developer portal](https://developer.okta.com/):

Reviewer 3: ```suggestion
If you need help during the submission process, use the Get Support section on the My App Integrations page after you sign in to the [OIN Manager](https://oinmanager.okta.com). This section provides the following resources from the [Okta developer portal](https://developer.okta.com/):
```

* OIN integration guides
* Okta, OIDC, SAML, and SCIM concepts
* A search tool to find articles in the Okta developer portal

> If you have questions or need additional support to publish your app integration, you can reach out to the Okta OIN team directly at <oin@okta.com>.

Reviewer 2: Do we need to include a mailto: type link?

You may also want to include information on how to contact your organization if the customer has any support queries.

## Submit an integration

To start your integration submission, open the [OIN Manager](https://oinmanager.okta.com) and click **Start Submission Form**.

Sign in using your development org credentials and click **Add New Submission** to create a new submission instance.

If you want to review an in-progress submission, click **View** beside the name of your integration.

> Start to define your submission by specifying details in the [General Settings](#configure-general-settings) and [protocol specific](#configure-protocol-specific-settings) tabs.

Reviewer 2: "Start to define your" > "Begin defining your.."/"Start defining your..."?

If you need to update an integration, see [Update your published integration](#update-your-published-integration).

### Configure general settings

> In the General Settings page, you need to fill in the basic information about your integration:

Reviewer: ```suggestion
On the General Settings page, fill in the basic information about your integration:
```
